### PR TITLE
Product list endpoint

### PR DESCRIPTION
**Pull request by Aegistov**
#### Changes
- Added `.idea` to `.gitignore` so I don't accidentally push that to repo
- Added a Readme for the project to describe how to run
- Added folder structure for resources/endpoints
- Added the product list endpoint
  - Added product reader class to separate read logic from api

#### Testing
- Manual testing `http://localhost:5000/v1/products`

### CODE REVIEW

File: android_16/resources/products/products_api.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+from android_16.resources.products.product import Product
+from android_16.resources.products.product_reader import ProductReader
+
+
+class ProductsApi(MethodView):
+    @staticmethod
+    def get():
+        resp = [Product.serialize(item) for item in ProductReader.get_products()]
+        print(f'MMOREL {resp}')
```
Comment (Aegistov) [inline android_16/resources/products/products_api.py]: Remove print

File: android_16/resources/products/products_api.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from flask.views import MethodView
+
+from android_16.resources.products.product import Product
+from android_16.resources.products.product_reader import ProductReader
+
+
+class ProductsApi(MethodView):
+    @staticmethod
+    def get():
+        resp = [Product.serialize(item) for item in ProductReader.get_products()]
```
Comment (Aegistov) [inline android_16/resources/products/products_api.py]: This logic should be handled by something other than the API. Create Api Actor class

File: android_16/resources/products/product_reader.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import os
+from csv import reader
+
+from android_16.resources.products.product import Product
+
+
+class ProductReader:
+    CSV_NAME = f'{os.getcwd()}/android_16/data_store/products.csv'
```
Comment (Aegistov) [inline android_16/resources/products/product_reader.py:8]: Seems a little hacky, but fine for now as we're eventually going to move from CSV to MySQL

File: android_16/resources/products/products_api.py (hunk 1 above)
Comment (Aegistov) [reply android_16/resources/products/products_api.py]: Done

File: android_16/resources/products/products_api.py (hunk 2 above)
Comment (Aegistov) [reply android_16/resources/products/products_api.py]: Moved logic

**Review by JMonk54 — APPROVED** [review]

**Pull request merged**